test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
It is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) in this project that the test.md file should be ignored by git😉.
<img width="391" alt="testgitignore" src="https://user-images.githubusercontent.com/64643995/211574450-585510a7-6728-41c2-8d6a-ed57a4c33f7d.png">
Your test.md file is still on GitHub, which means it is still not being ignored. kindly fix this issue as it is a vital requirement to complete this milestone. 🤗🤗